.github/workflows/ci.yaml

```diff
@@ -38,6 +52,32 @@ jobs:
       - name: Checkout submodules
         run: git submodule update --init --recursive
 
+      # Automatic setup cache(Doesn't work )
+      # - name: Setup Nix
+      #   if:  matrix.runner == 'ubuntu-latest' || matrix.runner == 'macos-latest'
+      #   uses: DeterminateSystems/nix-installer-action@main
+
+      # - name: Setup Magic Nix Cache
+      #   if:  matrix.runner == 'ubuntu-latest' ||  matrix.runner == 'macos-latest'
+      #   uses: DeterminateSystems/magic-nix-cache-action@main
+
+      # - name: Run Nix Flake Check
+      #   if:  matrix.runner == 'ubuntu-latest' ||  matrix.runner == 'macos-latest'
+      #   run: nix flake check
```

Comment:
Also, are these comments intented? May I ask you the purpose?

Reply:
Yes, I prefer use `magic nix cache` to manage CI cache, but I still haven't dealt with it so far.
 I'm not familiar with `nix`. Maybe someone can solve it in the future.

Reply:
Okay :)